.ansible-lint

```diff
@@ -0,0 +1,7 @@
+---
+exclude_paths:
+  - ./tasks/old_content
+x:
+  - '303'
```

Comment:
Could you add some comments here about why each is listed and what it means?

Reply:
This configuration is moved from https://github.com/MindPointGroup/RHEL7-CIS/blob/devel/molecule/default/molecule.yml#L22-L27
Molecule simplified the lint configuration to just running commands, so we should put this configuration into ansible-lint's default configuration file

Reply:
Don't run ansible-lint for files in these paths. This speeds up ansible-lint significantly.
```yaml
exclude_paths:
  - ./tasks/old_content
```

Don't check ansible-lint [rule 303](https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/CommandsInsteadOfModulesRule.py).
This avoids a lot of warnings about using rpm, e.g. https://github.com/MindPointGroup/RHEL7-CIS/blob/devel/tasks/prelim.yml#L32-L41
```yaml
x:
  - '303'
```

I removed rule exclusions where it was possible to fix the code:
* [306] (https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/ShellWithoutPipefail.py) - I've added the pipefail option where required.
* [602](https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/ComparisonToEmptyStringRule.py) - removed comparisons to empty string

Reply:
I've resolved everything for rules 306 and 602, so we can also remove the exclusions from the configuration.